clang/lib/Parse/ParseExpr.cpp

```diff
@@ -3999,6 +4008,23 @@ std::optional<AvailabilitySpec> Parser::ParseAvailabilitySpec() {
       Actions.CodeCompletion().CodeCompleteAvailabilityPlatformName();
       return std::nullopt;
     }
+
+    if (Tok.is(tok::identifier) && GetLookAheadToken(1).is(tok::equal)) {
```

Comment:
Can you make it `tok::colon` instead? I think it's too late to change the `feat:` one, and it would be nice if they matched.